tests/ui/pattern/bindings-after-at/borrowck-pat-ref-mut-and-ref.stderr

```diff
@@ -10,18 +10,18 @@ error: cannot borrow value as mutable more than once at a time
   --> $DIR/borrowck-pat-ref-mut-and-ref.rs:33:9
    |
 LL |     let ref mut a @ (ref b @ ref mut c) = u(); // sub-in-sub
-   |         ^^^^^^^^^    -----   --------- value is mutably borrowed by `c` here
-   |         |            |
-   |         |            value is borrowed by `b` here
+   |         ^^^^^^^^^   ------   --------- value is mutably borrowed by `c` here
+   |         |           |
+   |         |           value is borrowed by `b` here
```

Comment:
This is technically a regression that I haven't figured out how to get rid of without breaking anything else :-/